kommunicateui/src/main/res/values/strings.xml

```diff
@@ -227,4 +227,26 @@
     <string name="km_active_bots_text">Active bot:</string>
     <string name="km_default_assigned_bot_name">Support Bot</string>
     <string name="km_take_over_from_bot_button_text">Take over from bot</string>
+    <string name="resolve_info_text1">later</string>
+    <string name="resolve_info_text2">la</string>
```

Comment:
@shubhamtewari where is this 1, 2, 3 getting used?

Reply:
Nowhere actually, removed them